aesthetic.css

```diff
@@ -200,6 +203,11 @@ img {
   margin-bottom: 0.6rem;
 }
 
+a.logo img {
```

Comment:
This ensures that the default image spacing and 'circling' of images doesn't occur for the logo. To be honest, this will probably need to be refactored to be a class of its own as I start including more images that won't be rounded...